content/en/news/releases/1.7.x/announcing-1.7.1/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Announcing Istio 1.7.1
+linktitle: 1.7.1
+subtitle: Patch Release
+description: Istio 1.7.1 patch release.
+publishdate: 2020-09-10
+release: 1.7.1
+aliases:
+    - /news/announcing-1.7.1
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.7.0 and Istio 1.7.1
+
+{{< relnote >}}
+
+# Changes
+
+
+
+- **Added** Envoy [ext authz and gRPC access log API support](https://github.com/istio/istio/wiki/Enabling-Envoy-Authorization-Service-and-gRPC-Access-Log-Service-With-Mixer) in Mixer,
+which makes Mixer based configuration and out of process adapter still work after upgrading to future version of Istio. 
+  ([Issue #23580](https://github.com/istio/istio/issues/23580))
+
+
+
+- **Fixed** Remove unreachable endpoints for non-injected workloads across networks.
+  ([Issue #26517](https://github.com/istio/istio/issues/26517))
+
+- **Fixed** HoldApplicationUntilProxyStarts breaks rewriteAppProbers.
+  ([Issue #26873](https://github.com/istio/istio/issues/26873))
+
+- **Fixed** deleting the remote-secret for multicluster installation removes remote endpoints.
```

Comment:
Is there a Github issue for this?

Reply:
@stevenctl could you attach a issue id for PR https://github.com/istio/istio/pull/26899, and add it to the release note?

Reply:
I don't think there was an issue for this. I can retroactively create one if necessary.

Reply:
```suggestion
- **Fixed** deleting the remote-secret for multicluster installation removes remote endpoints.
  ([Issue #27187](https://github.com/istio/istio/issues/27187))
```